requirements/base.txt

```diff
@@ -6,26 +6,27 @@
 #
 certifi==2019.11.28       # via requests
 chardet==3.0.4            # via requests
-django-waffle==0.18.0     # via edx-django-utils
-django==1.11.26           # via edx-django-utils, rest-condition
+django-waffle==0.20.0
+django==2.2.11
```

Comment:
There should probably be a constraint keeping this < 2.0 until we start testing the newer versions in CI.